File: 07 - DAT203.2x - Principles of Machine Learning/Labs/Module 2/SetDayOfWeek.R
set.day.of.week <- function(){
  library(dplyr)
  ## First day in the dataset is Sunday
  data.frame(weekday= c(0, 1, 2, 3, 4, 5, 6),
             dayOfWeek = c("Sun", "Mon", "Tue", "Wed", "Thr", "Fri", "Sat"),
             stringsAsFactors = FALSE)  
}


df <- set.day.of.week() 
maml.mapOutputPort('df')

File: 07 - DAT203.2x - Principles of Machine Learning/Labs/Module 2/SetDayOfWeek.py
def day_of_week():  
    import pandas as pd
    ## First day in the dataset is Sunday
    days = pd.DataFrame([[0, 1, 2, 3, 4, 5, 6],
                        ["Sun", "Mon", "Tue", "Wed", "Thr", "Fri", "Sat"]]).transpose()
    days.columns = ['weekday', 'dayOfWeek']                        
    return days  
    
def azureml_main():
    return day_of_week()